i18n/en.json

```diff
@@ -4470,6 +4471,7 @@
   "tutorial_tip.ok": "Next",
   "tutorial_tip.out": "Opt out of these tips.",
   "tutorial_tip.seen": "Seen this before? ",
+  "ui": "Interested in our higher-security features?",
```

Comment:
Should update this message id to something more standardized?

Reply:
Thanks! I forgot these